### PR TITLE
Validate WHY

### PR DESCRIPTION
**Pull request by Wenwifhat**
# Validate [WHY](https://solscan.io/token/WHYoaBumcmxCqw38y2mjs4cVkCBgwiDizbMVvcejmGT)

## Attestations (Please provide links):
- Tweet from your Twitter Account attesting the Mint address, tagging [@JupiterExchange](https://twitter.com/JupiterExchange) and showing community support: https://x.com/wenwifhat_sol/status/1794627291118760192?s=46
- Coingecko URL: https://www.coingecko.com/en/coins/wenwifhat
- CMC URL: https://coinmarketcap.com/currencies/wenwifhat/

## Validation (Please check off boxes):
- [x] The metadata provided in the PR matches what is on-chain (Mandatory)
- [x] Does not duplicate the symbol of another token on Jupiter's strict list (If not, review will be delayed)
- [x] Is Listed on Coingecko / CMC (Optional, but helpful for reviewers)  

## Acknowledgement (Please check off boxes)
- [x] My change matches the format in the file (no spaces between fields).
- [x] My token is already live and trading on Jupiter.
- [x] !!! I read the README section on Community-Driven Validation and understand this PR will be only be reviewed when there is community support on Twitter.
- [x] Please make sure your pull request title has your token name. If it just says "Main", or "Validate", it will automatically be closed. PRs containing broken attestation or solscan links will also be closed.

### CODE REVIEW

File: validated-tokens.csv
Comment (knobfly) [inline validated-tokens.csv:1]: Validate why!

**Comment by charcutier22** [comment]
This will be great for the #solana blockchain!

**Comment by JoFo44** [comment]
$WHY

**Comment by Dannyboy241** [comment]
I can hereby attest that the provided information are accurate, and the token is valid.

 - [x] Mint address: WHYoaBumcmxCqw38y2mjs4cVkCBgwiDizbMVvcejmGT
 - [x] Symbol: WHY
 - [x] Token Decimals: 9
 - [x] Token icon: https://zusqmjtp6ujez6gm5rxjzouetdesm2yx2rotyroljao25ri4ntda.arweave.net/zSUGJm_1Ekz4zOxunLqEmMkmaxfUXTxFy0gdrsUcbMY
 - [x] On-chain metadata matches
 - [x] No duplication on Jupiter's strict list

**Comment by Beer111** [comment]
I can hereby attest that the provided information are accurate, and the token is valid.

 - [x] Mint address: WHYoaBumcmxCqw38y2mjs4cVkCBgwiDizbMVvcejmGT
 - [x] Symbol: WHY
 - [x] Token Decimals: 9
 - [x] Token icon: https://zusqmjtp6ujez6gm5rxjzouetdesm2yx2rotyroljao25ri4ntda.arweave.net/zSUGJm_1Ekz4zOxunLqEmMkmaxfUXTxFy0gdrsUcbMY
 - [x] On-chain metadata matches
 - [x] No duplication on Jupiter's strict list

**Comment by Solplane** [comment]
Validate Wenwifhat this will be a top project on Solana

**Comment by Moonwalker707** [comment]
Validate $WHY 
Wenwifhat is here to stay

**Comment by xizaniels** [comment]
I can hereby attest that the provided information are accurate, and the token is valid.

 - [x] Mint address: WHYoaBumcmxCqw38y2mjs4cVkCBgwiDizbMVvcejmGT
 - [x] Symbol: WHY
 - [x] Token Decimals: 9
 - [x] Token icon: https://zusqmjtp6ujez6gm5rxjzouetdesm2yx2rotyroljao25ri4ntda.arweave.net/zSUGJm_1Ekz4zOxunLqEmMkmaxfUXTxFy0gdrsUcbMY
 - [x] On-chain metadata matches
 - [x] No duplication on Jupiter's strict list

File: validated-tokens.csv
Comment (knobfly) [reply validated-tokens.csv:1]: I can hereby attest that the provided information are accurate, and the token is valid.

 - [x] Mint address: WHYoaBumcmxCqw38y2mjs4cVkCBgwiDizbMVvcejmGT
 - [x] Symbol: WHY
 - [x] Token Decimals: 9
 - [x] Token icon: https://zusqmjtp6ujez6gm5rxjzouetdesm2yx2rotyroljao25ri4ntda.arweave.net/zSUGJm_1Ekz4zOxunLqEmMkmaxfUXTxFy0gdrsUcbMY
 - [x] On-chain metadata matches
 - [x] No duplication on Jupiter's strict list

**Comment by Jymfletzfur** [comment]
insufficient volume

**Comment by Wenwifhat** [comment]
we originally applied  3 weeks ago we had 200k in volume . Our volume 
will come for now I have been focused on wenpad.io and building 
sustainability on 0 tax

On 2024-05-26 23:11, Jymfletzfur wrote:

> insufficient volume
> 
> --
> Reply to this email directly, view it on GitHub [1], or unsubscribe 
> [2].
> You are receiving this because you authored the thread.Message ID: 
> ***@***.***>



Links:
------
[1] 
https://github.com/jup-ag/token-list/pull/3981#issuecomment-2132411772
[2]

**Pull request closed without merging**